app/views/pages/home/_signin.html.haml

```diff
@@ -1,11 +1,16 @@
 %h2.govuk-heading-m= t('.title')
 
-%p= t('.description')
+- if authenticated?
+  %div.manage-vacancies
```

Comment:
no need to add `%div`

`.manage-vacancies` is enough to generate a div with that class